src/arfsdao.ts

```diff
@@ -793,15 +793,14 @@ export class ArFSDAO extends ArFSDAOAnonymous {
 	async assertValidPassword(password: string): Promise<void> {
 		const wallet = this.wallet;
 		const walletAddress = await wallet.getAddress();
-		const query = buildQuery(
-			[
+		const query = buildQuery({
+			tags: [
 				{ name: 'Entity-Type', value: 'drive' },
 				{ name: 'Drive-Privacy', value: 'private' }
 			],
-			undefined,
-			walletAddress,
-			DESCENDENT_ORDER
-		);
+			owner: walletAddress,
+			sort: ASCENDING_ORDER
```

Comment:
Isn't the default ASCENDING? If not, we DO have to change it at `buildQuery()` args